clang/include/clang/Serialization/ASTReader.h

```diff
@@ -696,7 +696,7 @@ class ASTReader
   /// Mapping from global submodule IDs to the module file in which the
   /// submodule resides along with the offset that should be added to the
   /// global submodule ID to produce a local ID.
-  GlobalSubmoduleMapType GlobalSubmoduleMap;
+  mutable GlobalSubmoduleMapType GlobalSubmoduleMap;
```

Comment:
Is `mutable` necessary here?

Reply:
Oh, I don't know why it remains. Maybe I added during the work progress.